inginious/common/field_types.py

```diff
@@ -16,3 +16,7 @@ class FieldTypes(Enum):
     INTEGER = 1
     STRING = 2
     BOOLEAN = 3
+
+    def get_cast_type(self):
```

Comment:
You should define a default value for these types, so that we know what to send to the backend if there is no value set by the user yet.

Maybe it would be worth replacing the enum values by a namedtuple ``(id, class, default_value)`` and storing ``FieldType.XXXX.value.id`` in yaml, so that the code is less error-prone.